views/random-beer.hbs

```diff
@@ -0,0 +1,14 @@
+
+<div class = "random-beers">
+    <ul class="beer-list2">
+    {{#each responseFromAPI }}
+        <h2>{{this.name}}</h2>
+        <img src="{{this.image_url}}" alt="{{this.tagline}}">
+        <p>{{this.tagline}}</p>
+        {{#each this.food_pairing}}
+        <li>{{this}}</li>
```

Comment:
be carefull with this line, you are printing the entire object, instead of refering any special Key-( like name, or image_url)